include/ck_tile/ops/gemm/pipeline/gemm_pipeline_problem.hpp

```diff
@@ -3,72 +3,69 @@
 
 #pragma once
 
-#include "ck_tile/core.hpp"
 #include "ck_tile/ops/gemm/pipeline/gemm_pipeline_ag_bg_cr_scheduler.hpp"
 
 namespace ck_tile {
 
 static constexpr int _VectorSize = 16;
 
-template <typename ADataType_,
-          typename BDataType_,
-          typename CDataType_,
-          typename BlockGemmShape_,
-          typename TileGemmTraits_>
+template <typename TileGemmTraits_>
```

Comment:
The `GemmPipelineProblem<>` & `UniversalGemmPipelineProblem<>` become simple wrappers. Do we really need them now?

Reply:
Put the data type and gemm shape back to the pipeline problem.